api_app/db/repositories/workspaces.py

```diff
@@ -36,8 +36,11 @@ def get_deployed_workspace_by_workspace_id(self, workspace_id: str) -> Workspace
         return workspace
 
     def get_workspace_by_workspace_id(self, workspace_id: str) -> Workspace:
```

Comment:
Similar to the change in api_app/api/dependencies/workspaces.py
Maybe it is worth also renaming it here to : get_workspace_by_id ?

Reply:
@LizaShak Fixed -- let me know if there is something else or if you are good to approve.